src/app/socket/adminSocket/admin.socket.ts

```diff
@@ -17,13 +18,14 @@ export class AdminSocket extends DefaultSocket {
     }
   }
 
-  addQuestion(question: Question){
-    this.getStompClient().send("/admin/question/add", {}, question.text);
-  }
-
   public onQuestion(): Observable<Question>
   {
     return this.subscribe<Question>('/admin/question')
   }
 
+  public onJoinParticipant(sessionId: number): Observable<Participant>
```

Comment:
Würde die Methode umbenennen, da es irrelevant ist, dass der Joinende ein participant ist.

Reply:
Nein, in diesem Falls beschreibt die Methode genau das was hier passiert. Es wird ein Objekt mit dem Namen Participant übergeben. Im Aufbau des Systems wird es keine andere Benachrichtigungen geben.